integration-tests/actions/ocr_helpers.go

```diff
@@ -228,16 +228,14 @@ func CreateOCRJobs(
 				Name: nodeContractPairID,
 				URL:  fmt.Sprintf("%s/%s", mockserver.Config.ClusterURL, strings.TrimPrefix(nodeContractPairID, "/")),
 			}
-			// only create the bridge on the first loop of the nodes
```

Comment:
So we do now want to create multiple of the same bridge? This caused issues for us a while back in soak tests.

Reply:
Oh they aren't identical anymore so it won't cause an issue.